dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+   respond with an HTTP status code "200" and a JSON payload containing
+   a JSON Resource Descriptor (JRD) that contains a link conforming to
+   the JRD section (Section 4) of this document.
+
+   For example, a PayID server might respond to a PayID Discovery query
+   with the following payload:
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 4]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+    {
+       "subject": "payid:alice$wallet.com",
+       "links": [
+           {
+               "rel" : "https://payid.org/ns/payid-easy-checkout/1.0",
+               "href": "https://wallet.com/checkout"
+           }
+       ]
+    }
+
+   A PayID Discovery client MUST parse this response to find the PayID
+   Easy Checkout Link.  If the JRD returned from the PayID Discovery
+   query does not contain a PayID Easy Checkout Link in its 'links'
+   collection, PayID Easy Checkout is considered to have failed.  Once a
+   PayID Easy Checkout URL has been obtained from the PayID Easy
+   Checkout Link, PayID Easy Checkout Discovery is considered to be
+   complete.
+
+3.2.  PayID Easy Checkout URL Assembly
+
+   A PayID Easy Checkout URL represents the resource on a wallet that
+   can be used by a sender to complete a payment.  However, before
+   directing a payer to their sending client, the payment receiver MUST
+   append all of the query parameters defined in the following section
+   (Section 3.2.1).
+
+   Once a PayID Easy Checkout URL is assembled, PayID Easy Checkout is
+   considered to be complete.
+
+3.2.1.  PayID Easy Checkout URL Query Parameters
+
+   This specification defines several query parameter names and
+   corresponding datatypes which MUST be added to the PayID Easy
+   Checkout URL before redirecting a payer to their wallet client.  The
+   PayID Easy Checkout URL SHOULD be parsed by the wallet in order to
+   retrieve any values set by the payment recipient.  It is RECOMMENDED
+   that wallets use these values to pre-populate a payment transaction.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 5]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+   +----------------+---------+----------------------------------------+
+   | Name           | Type    | Description                            |
+   +----------------+---------+----------------------------------------+
+   | amount         | integer | The amount that should be sent by the  |
+   |                |         | sender to the receiver                 |
+   |                |         |                                        |
+   | receiverPayID  | string  | The (PayID URI)[PAYID-URI] of the      |
+   |                |         | receiver                               |
+   |                |         |                                        |
+   | assetCode      | string  | The ISO-4217 currency code that        |
```

Comment:
Reference to ISO-4217

Reply:
I'm actually getting rid of ISO-4217 and replacing with a reference to the PayID Protocol defined currency codes